sw/device/tests/entropy_src_smoketest.c

```diff
@@ -16,8 +16,8 @@ const test_config_t kTestConfig;
 const size_t kEntropyDataNumWords = 12;
 
 const uint32_t kExpectedEntropyData[] = {
-    0xa8f49c0d, 0x148ca619, 0xd1818b93, 0x25f2397d, 0x32955611, 0x0aca4b8e,
-    0xc0956655, 0x80735507, 0x4cf2b852, 0x97e50e09, 0x39649525, 0x6a2795f0,
+    0xdaf90306, 0x0466d674, 0x9691df54, 0xd2e3c93f, 0xdcbc993e, 0x64bfd173,
```

Comment:
Were does this come from if there is no LSFR now?

Reply:
the ast, which is why this could not be done until i did a pile of other PRs to get ast into verilator and fpga.